modules/mobile/app/controllers/mobile/v0/community_care_providers_controller.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class CommunityCareProvidersController < ApplicationController
```

Comment:
I chose to go with the full name "CommunityCare" because I thought it was clearer. I know we use "cc" a lot in appointments, but I think there is value in being more explicit. The web controller is just called ccp_controller, which does not make it any easier to find if you're not familiar with it.

Reply:
I agree this is the way to go. I changed my PR to match this.